docs/en/02_Developer_Guides/14_Files/07_File_Usage.md

```diff
@@ -0,0 +1,63 @@
+---
+title: File usage
+summary: See file usage and customising the file "Used on" table
+icon: compress-arrows-alt
+---
+
+# File Usage
+
+In the Files section of the CMS, click on a file to see file details.  Within the file details panel there is a "Used on" tab that shows a table of Pages and other DataObjects where the file is used throughout the website.
+
+## Customising the File "Used on" table in the Files section (asset-admin)
+
```

Comment:
Maybe add a blurb to contextualise this a bit.

```suggestion

Your project specific DataObject will automatically be displayed on the "Used on". This may not always be desirable, especially when working with background DataObjects the user can not interact with directly. Extensions can be applied to the `UsedOnTable` class to suppress specific entries.

```

Reply:
Added